HopsanGUI/Utilities/HighlightingUtilities.cpp

```diff
@@ -348,60 +354,74 @@ CppHighlighter::CppHighlighter(QTextDocument *parent)
                     << "\\bslots\\b" << "\\bstatic\\b" << "\\bstruct\\b"
                     << "\\btemplate\\b" << "\\btypedef\\b" << "\\btypename\\b"
                     << "\\bunion\\b" << "\\bunsigned\\b" << "\\bvirtual\\b"
-                    << "\\bvoid\\b" << "\\bvolatile\\b" << "\\busing\\b";
-    foreach (const QString &pattern, keywordPatterns) {
+                    << "\\bvoid\\b" << "\\bvolatile\\b" << "\\busing\\b"
```

Comment:
Should this list really contain Qt keywords, foreach and slots (not that it really matters). Also constexpr should be added. (C++11 so cant be used in hopsan components... well it can if a user make own components and want c++11)

Reply:
I removed the Qt keywords and added constexpr.